gax-java/gax-bom/pom.xml

```diff
@@ -48,6 +48,7 @@
         <groupId>com.google.api</groupId>
         <artifactId>gax</artifactId>
         <version>2.23.3-SNAPSHOT</version><!-- {x-version-update:gax:current} -->
+        <type>test-jar</type>
```

Comment:
@JoeWang1127  ` 'dependencies.dependency.version' for com.google.api:gax:jar:testlib is missing. @ line 80, column 21` came from the BOM's declaration of the artifacts. Do you really need that?

Reply:
A BOM doesn't determine what code goes to JAR file.

Reply:
There's an error if I delete this line and reload the project:
```
'parent.relativePath' of POM com.google.api:gax-bom:2.23.3-SNAPSHOT (/Users/joewa/Documents/workspace/gapic-generator-java/gax-java/gax-bom/pom.xml) points at com.google.api:gax-parent instead of com.google.cloud:google-cloud-shared-config, please verify your project structure
```

Reply:
OK it’s for IntelliJ. Then it makes sense.